docs/examples/create_account_iam_resources.md

```diff
@@ -0,0 +1,142 @@
+
+~> **NOTE:** This example uses "Red Hat Cloud Services Provider" (this provider) to get a list of all OCP supported versions, this list need to be added to the "rosa-sts" module apply, in order to validate the requested version. Also it used to get the list of all permissions required for policy creation.
+
+## Input Variables (variables.tf)
+
+This example include some variables which need to be added before running `terraform apply`
+Terraform requires a value for every variable. There are several ways to assign variable values. see [Assigning Values to Root Module Variables](https://developer.hashicorp.com/terraform/language/values/variables#assigning-values-to-root-module-variables) for more info.
+
+```terraform
+variable "ocm_environment" {
+  type    = string
+  default = "production"
+}
+
+variable "openshift_version" {
+  type = string
+  default = ""
+}
+
+variable "account_role_prefix" {
+  type    = string
+  default = ""
+}
+
+variable "token" {
+  type = string
+}
+
+variable "url" {
+  type        = string
+  description = "Provide OCM environment by setting a value to url"
+  default     = "https://api.openshift.com"
+}
+
+variable "path" {
+  description = "(Optional) The arn path for the account/operator roles as well as their policies."
+  type        = string
+  default     = null
+}
+
+variable "tags" {
+  description = "List of AWS resource tags to apply."
+  type        = map(string)
+  default     = null
+}
+```
+
+## Terraform output (output.tf)
+
+```terraform
+output "account_role_prefix" {
+  value = module.create_account_roles.account_role_prefix
+}
+```
+
+## Provider decleration (main.tf)
+
+```terraform
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = ">= 4.20.0"
+    }
+    rhcs = {
+      source  = "terraform-redhat/rhcs"
+    }
+  }
+}
+
+provider "rhcs" {
+  token = var.token
+  url   = var.url
+}
+```
+
+## Usage example (iam_resources.tf)
+
+```terraform
+data "rhcs_policies" "all_policies" {}
+
+data "rhcs_versions" "all" {}
+
+module "create_account_roles" {
+  source  = "terraform-redhat/rosa-sts/aws"
+  version = "0.0.12"
+
+  create_operator_roles = false
+  create_oidc_provider  = false
+  create_account_roles  = true
+
+  account_role_prefix    = var.account_role_prefix
+  ocm_environment        = var.ocm_environment
+  rosa_openshift_version = var.openshift_version
+  account_role_policies  = data.rhcs_policies.all_policies.account_role_policies
+  operator_role_policies = data.rhcs_policies.all_policies.operator_role_policies
+  all_versions           = data.rhcs_versions.all
+  path                   = var.path
+  tags                   = var.tags
+}
+```
+
+## Verification
+
+1. In the `rosa` CLI, run the following command to verify that the account roles are created:
```

Comment:
probably users would be interested to verify also from AWS side, but I didn't find the link with the description in AWS docs.